packages/inquirer-common/package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "@sap-ux/inquirer-common",
+    "description": "Commonly used functionality to support inquirer modules",
+    "version": "0.1.0",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/SAP/open-ux-tools.git",
+        "directory": "packages/inquirer-common"
+    },
+    "bugs": {
+        "url": "https://github.com/SAP/open-ux-tools/issues?q=is%3Aopen+is%3Aissue+label%3Abug+label%3Ainquirer-common"
+    },
+    "license": "Apache-2.0",
+    "main": "dist/index.js",
+    "scripts": {
+        "build": "tsc --build",
+        "clean": "rimraf --glob dist test/test-output coverage *.tsbuildinfo",
+        "watch": "tsc --watch",
+        "lint": "eslint . --ext .ts",
+        "lint:fix": "eslint . --ext .ts --fix",
+        "test": "jest --ci --forceExit --detectOpenHandles --colors --passWithNoTests",
+        "test-u": "jest --ci --forceExit --detectOpenHandles --colors -u",
+        "link": "pnpm link --global",
+        "unlink": "pnpm unlink --global"
+    },
+    "files": [
+        "LICENSE",
+        "dist",
+        "templates",
+        "!dist/*.map",
+        "!dist/**/*.map"
+    ],
+    "dependencies": {
+        "@sap-ux/ui5-info": "workspace:*",
+        "@sap-devx/yeoman-ui-types": "1.14.4",
+        "fuzzy": "0.1.3",
+        "i18next": "23.5.1",
+        "chalk": "4.1.2",
+        "figures": "3.2.0"
```

Comment:
This version is already 4 years old, could you please try using the latest version (I think `6.0.1`)

Reply:
So I took the versions from `inquirer@8.2.6`, where I took the `Separator`  implementation from (aligns with `@types/inquirer@8.2.6`). I would be concerned about compatibility with more recent versions. `figures@6.0.1` is an ESM module which is going to cause some issues.

Reply:
makes sense, didn't know that, I retract my ask.